app/src/main/java/de/tum/in/tumcampusapp/database/migrations/Migration3to4.kt

```diff
@@ -10,6 +10,7 @@ import android.arch.persistence.room.migration.Migration
  */
 class Migration3to4 : Migration(3, 4) {
     override fun migrate(database: SupportSQLiteDatabase) {
+        Migration2to3().migrate(database)
```

Comment:
Can we really do this unconditionally? What happens on clean installing 1.5.2, then updating to 1.5.3 with that mirgation?